docs/Coiote_IoT_DM/multi_firmware_update.md

```diff
@@ -0,0 +1,149 @@
+---
+title: Multi-component Firmware Update
+date: 2023-06-05
+og_title: AVSystem IoT Developer Zone
+---
+
+# Multi-component Firmware Update
+
+Remotely update a component of your LwM2M device's firmware using the **Advanced Firmware Update Object** `/33629`.
+
+The Advanced Firmware Update Object supports multiple instances, each representing a "component" of the device's firmware that can be upgraded separately. The specific meaning and purpose of these components are not standardized and can vary depending on the implementation. However, they typically encompass elements such as bootloaders, application code, cellular modem firmwares, security processor firmwares, and other related firmware entities.
+
+## Prerequisites
+
+- An active [Coiote DM](https://eu.iot.avsystem.cloud/) user account
+- A device which supports the Advanced Firmware Update Object `/33629`
+    
+## Firmware Update Object `/33629`
```

Comment:
I'd consider adding some note about how object 33629 is on-standard for now, but designed as an extension of object 5, etc. Because for now, the reader will not know where does this object come from and where to find additional information on it.

Mentioning that it is designed as an extension of object 5 would also be reassuring for those who are already familiar with that, so that they can easily note how e.g. the set of available update results is an extension of what's defined for object 5.